web/js/jquery.jmri.js

```diff
@@ -1026,16 +1022,33 @@
                     jmri.train(e.data.name, e.data);
                 },
                 trains: function (e) {
-                    jmri.trains(e.data);
+                    jmri.trains(e);
                 },
                 turnout: function (e) {
                     jmri.turnout(e.data.name, e.data.state, e.data);
                 },
                 turnouts: function (e) {
-                    jmri.turnouts(e.data);
+                    jmri.turnouts(e);
```

Comment:
Have these functions not worked before? I'm not against the change, just checking.

Reply:
None of the list functions have worked since Randall removed the code which called them 5 years ago.

Reply:
Thanks for fixing this.